backend/app/tests/utils/test_streaming_jsonl_extractor.py

```
@@ -0,0 +1,222 @@
+
+    def test_create_extractor(self):
+        """测试创建提取器"""
+        extractor = create_streaming_jsonl_extractor()
+        assert isinstance(extractor, StreamingJSONLExtractor)
+        assert extractor.state == ExtractionState.WAITING_FOR_JSON
+        assert extractor.pure_jsonl_content == ""
+
+    def test_basic_jsonl_extraction(self):
+        """测试基本JSONL提取"""
+        extractor = StreamingJSONLExtractor()
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)